news/1663-feature.md

```diff
@@ -0,0 +1 @@
+Added field UserName to ExtractionRequestInfoMessage
```

Comment:
Could you also mention the addition to the MongoDB schema, please?

Reply:
Updated in commit [ca5174f](https://github.com/SMI/SmiServices/pull/1663/commits/ca5174fc1a64b6f8bea53e25fb42544ef9fee730), is that what you were thinking of?

Reply:
Thanks!